### PR TITLE
Upgrade to python 3.11

### PR DESCRIPTION
**Pull request by diox**
Fixes mozilla/addons#1965

### CODE REVIEW

**Comment by diox** [comment]
You can test this by... well... doing anything in the container really. CI has been updated to run with the new version.

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
         'Framework :: Django',
         'Topic :: Internet :: WWW/HTTP :: Browsers',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.10',
```
Comment (KevinMind) [inline setup.py:28]: See above, if PYTHON_VERSION is specified in the container, you can substitute here, maybe it is defined in settings? or straight up environment.

File: Dockerfile [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # Read the docs/topics/development/docker.md file for more information about this Dockerfile.
 ####################################################################################################
 
-FROM python:3.10-slim-bookworm as olympia
```
[the remark this answers is not in the document]
Comment (KevinMind) [reply Dockerfile:5]: If we'd like to keep it hardcoded, could you remove the input from the build action (it's been there for a while with nothing to do)

File: setup.py (hunk 1 above)
Comment (diox) [reply setup.py:28]: Well since we're not publishing the package anyway I'm thinking of completely removing this classifier actually. It shouldn't matter.

**Review by KevinMind — APPROVED** [review]
LGTM

**Pull request merged**